tensorflow/core/kernels/conv_ops.cc

```diff
@@ -697,7 +697,11 @@ void LaunchConv2DOp<GPUDevice, T>::operator()(
   // in NHWC data layout. In all other configurations it's more efficient to
   // run computation in NCHW data format.
   const bool compute_in_nhwc =
+#if GOOGLE_CUDA
       DataTypeToEnum<T>::value == DT_HALF && IsVoltaOrLater(*stream->parent());
+#else
+      false;
+#endif
```

Comment:
Other convolution kernels in TF (conv_grad_filter_ops, conv_grad_input_ops) also have this variable defined in the same way. It seems this should be extracted into a function, and this change duplicated across those kernels?